packages/components/src/internal/components/auditlog/models.ts

```diff
@@ -65,6 +68,7 @@ export class TimelineEventModel extends Record({
     metadata: undefined,
     oldData: undefined,
     newData: undefined,
+    originalValues: undefined, // map from field name to user-provided values that were converted
```

Comment:
minor: this comment helps, but when I original saw the prop name of `originalValues` I thought that it was for the old value map...but that is `oldData`. I'm not sure that `convertedValues` or `userProvidedValues` would make it any clearer, so no real good suggestions.